cmake/configure_linux.intel.cmake

```diff
@@ -1,37 +0,0 @@
-message("")
```

Comment:
I don't see any cleanup of `cmake/configure_macosx.*.cmake`, and several others are also unchanged - is this intentional?

Reply:
Yes. I do not have macosx to test, so I left it as it is. It should be cleaned up by somebody who can run a test on mac. Also other config files (cheyenne, gaea, stampede) should be updated.

Reply:
So should we merge it and then commit a cleanup PR for the other platforms within a day, while Moorthi is working on his PR? These changes will be totally independent, won't affect Moorthi's work and any of the tier-1 platforms.

Reply:
What is the long-term strategy for keeping unsupported platforms?
This should not be a 2 step process where supported platforms are updated in one PR and then the unsupported in another.  This is bound to be unmaintainable.

We either remove the unsupported ones from this repo and make that a responsibility of the org that is supporting these platforms.  PR's should not be held up for platforms that developers have no access to.  NOAA Tier-1 platforms is a must.

Reply:
These platforms are not unsupported, they are tier-2 and tier-3. I am absolutely against removing the configurations that we use for daily work (How many real-time parallels are running on jet?! How much development for CMEPS etc is done on Cheyenne?! DTC relies on generic macOS and Linux platforms to work to do their work, and the community will too). The tier-1,2,3 concept makes sense as defined - tier-1 testing always pre-commit, tier-2/3 ideally too, but if need be retrospectively. I'd rather hold back this PR for another few hours until I tested the remaining platforms.

Reply:
Yes. These platforms are supported. They are just not in "Tier-1" category, meaning we do not require full regression test to pass before each commit. But we try our best to make sure model is running on those platforms, by providing libraries, etc. That support is not done by EMC, because we do not have access to those machines.

Reply:
Ok. I understand the concept of tier 1-2-3.
What is the agreed wait period for tier 2-3 platform to be tested. 1 week? Surely that is enough. Who is responsible for that?
What if there is an issue? Who will resolve the issues and how?
Is this information documented somewhere? 
Seems like we are locked in this type of conversation often.

I am not suggesting removing generic configs.  Those are necessary for development on generic platforms.

Reply:
Have a look here https://github.com/ufs-community/ufs-weather-model/wiki/Regression-Test-Policy-for-Weather-Model-Platforms-and-Compilers and here https://github.com/ufs-community/ufs/wiki/Regression-Test-Policy-for-UFS-Platforms-and-Compilers (the second is referenced in the first). Obviously still work in progress to some extend.